pkgs/applications/misc/onagre/default.nix

```
@@ -5,31 +5,40 @@
 , pkgconf
 , freetype
 , expat
+, libX11
+, libXcursor
+, libXi
+, libXrandr
+, pop-launcher
+, makeWrapper
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "onagre";
-  version = "1.0.0-alpha.0";
+  version = "1.0.0";
 
   src = fetchFromGitHub {
     owner = "oknozor";
     repo = pname;
```

```suggestion
    repo = "onagre";
```

https://github.com/NixOS/nixpkgs/issues/277994